backend/open_api_spec.yml

```diff
@@ -0,0 +1,753 @@
+          $ref: "#/components/responses/404"
+
+  /curation/collections/{collection_uuid}/datasets/upload-s3:
+    put:
+      summary: Get credentials for uploading local files
+      description: >
+        Retrieve temporary AWS credentials for uploading Dataset source files to S3 to create
+        or update Datasets for the specified Collection.
+      security:
+        - cxguserCookie: []
+        - curatorAccess: []
+      parameters:
+        - $ref: "#/components/parameters/path_collection_uuid"
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  credentials:
+                    type: object
+                    properties:
+                      access_key:
+                        type: string
+                      secret_access_key:
+                        type: string
+                      session_token:
+                        type: string
+        "401":
+          $ref: "#/components/responses/401"
+
+  /curation/collections/{collection_uuid}/datasets/status:
+    get:
+      summary: Get a Dataset's processing status
+      description: Get the processing status for a Dataset.
+      security:
+        - cxguserCookie: []
+        - curatorAccess: []
+      parameters:
+        - $ref: "#/components/parameters/path_collection_uuid"
+        - $ref: "#/components/parameters/query_curator_tag"
+        - $ref: "#/components/parameters/query_dataset_uuid"
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                    enum: [FAILURE, PENDING, SUCCESS]
+                  file_statuses:
+                    type: object
+                    properties:
+                      cxg:
+                        description: status for cxg conversion
+                        $ref: "#/components/schemas/conversion_status"
+                      h5ad:
+                        description: status for h5ad conversion
+                        $ref: "#/components/schemas/conversion_status"
+                      rds:
+                        description: status for rds/seurat conversion
+                        $ref: "#/components/schemas/conversion_status"
+
+  /curation/collections/{collection_uuid}/asset:
+    get:
+      summary: Retrieve links for downloading a dataset
+      description: >-
+        Request to download a file which on success will generate and return a pre-signed URL to download the dataset.
+      parameters:
+        - $ref: "#/components/parameters/path_collection_uuid"
+        - $ref: "#/components/parameters/query_curator_tag"
+        - $ref: "#/components/parameters/query_dataset_uuid"
+        - $ref: "#/components/parameters/query_visibility"
```

Comment:
Added visibility so that the curator can specify which dataset they want to download.